### ISSUE TITLE
Tooltips should be freely positionable and stay visible until mouse leaves widget

### ISSUE THREAD

**Issue by sinfricia** [jtackaberry/rtk#14]
With the current implementation tooltips flicker with shorter tooltip delay times.
Or maybe it could be implemented as something like 'hide tooltip after mouse has moved a certain distance'.

In combination with the possibility to specify the position of the tooltip this would expand the capabilities of tooltips by a lot.

**Comment by jtackaberry** [comment]
I agree the current behavior is kinda janky, and a distance threshold makes sense.  Will look at this for rtk v1.4.
